### PR TITLE
Chore/clean up

### PR DESCRIPTION
**Pull request by 00xSam**
- Refactored; remove `position(V1)` interaction from SDK
- Throw error in `removeLiquidity` function if position doesn't have any liquidity 
- Added Function `getPosition` to retrieve a single position data

### CODE REVIEW

File: ts-client/src/dlmm/index.ts [hunk 1]
```diff
@@ -2007,6 +1646,78 @@ export class DLMM {
...
+        lowerBinArrayPubKey,
+        upperBinArrayPubKey,
+      ]
+    );
+
+    const onChainTimestamp = new BN(
+      clockAccInfo.data.readBigInt64LE(32).toString()
+    ).toNumber();
+
+    const lowerBinArray = this.program.coder.accounts.decode(
```
Comment (andrewsource147) [inline ts-client/src/dlmm/index.ts:1694]: Would have the case that position is initialized, but lowerBinArray is not initialized
So position would have zero liquidity

File: ts-client/src/dlmm/index.ts [hunk 2]
```diff
@@ -2007,6 +1646,78 @@ export class DLMM {
...
+      "binArray",
+      lowerBinArrayAccInfo.data
+    )
+    const upperBinArray = this.program.coder.accounts.decode(
```
Comment (andrewsource147) [inline ts-client/src/dlmm/index.ts:1698]: ditto^

File: ts-client/src/dlmm/index.ts [hunk 3]
```diff
@@ -2952,12 +2698,43 @@ export class DLMM {
...
+    const upperBinIdToRemove = Math.max(...binIds);
+    const [lowerBinArrayPubKey] = deriveBinArray(this.pubkey, binIdToBinArrayIndex(new BN(binIds[0])), this.program.programId)
+    const [upperBinArrayPubKey] = deriveBinArray(this.pubkey, binIdToBinArrayIndex(new BN(binIds[binIds.length - 1])), this.program.programId)
+    const [positionAccInfo, lbPairAccInfo, upperBinArrayInfoAcc, lowerBinArrayInfoAcc] = await this.program.provider.connection.getMultipleAccountsInfo([
+      position,
+      this.pubkey,
+      upperBinArrayPubKey,
+      lowerBinArrayPubKey,
+    ])
+    const { lbPair, owner, feeOwner, upperBinId: positionUpperBinId, lowerBinId: positionLowerBinId } = this.program.coder.accounts.decode(
```
Comment (codewithgun) [inline ts-client/src/dlmm/index.ts:2711]: No need to fetch for `lbPair`, `lower + upper` bin array to check if a position is empty. You just need `position` account and check `liquidityShares` are zero

Comment (00xSam) [reply ts-client/src/dlmm/index.ts]: some state from lbPair still needed to pass to program interaction accounts param, removed lower + upper 👍

Comment (codewithgun) [reply ts-client/src/dlmm/index.ts]: This function is non-static, the `lbPair` state should already cached in the instance

File: ts-client/src/dlmm/index.ts (hunk 1 above)
Comment (00xSam) [reply ts-client/src/dlmm/index.ts:1742]: pushed a fix where if no bin array it will return position data with zero liquidity

**Review by andrewsource147 — APPROVED** [review]

**Pull request merged**